repositories/src/main/scala/cromwell/pipeline/datastorage/dto/project/configuration/ProjectConfigurationRequest.scala

```diff
@@ -0,0 +1,10 @@
+package cromwell.pipeline.datastorage.dto.project.configuration
+
+import cromwell.pipeline.datastorage.dto.{ProjectFileConfiguration, ProjectId}
+import play.api.libs.json.{Json, OFormat}
+
+case class ProjectConfigurationRequest(projectId: ProjectId, projectFileConfigurations: List[ProjectFileConfiguration])
+
+object ProjectConfigurationRequest {
+  implicit val projectConfigurationRequestFormat: OFormat[ProjectConfigurationRequest] = Json.format[ProjectConfigurationRequest]
+}
```

Comment:
Turn on `Ensure an empty line at the end of a file on Save` to get rid of these red warnings.

Reply:
Done